src/sentry/snuba/metrics/naming_layer/mapping.py

```diff
@@ -63,12 +68,22 @@ def get_public_name_from_mri(internal_name: Union[TransactionMRI, SessionMRI, st
         internal_name = internal_name.value
     assert isinstance(internal_name, str)
 
-    try:
+    if internal_name in MRI_TO_NAME:
         return MRI_TO_NAME[internal_name]
-    except KeyError:
+    elif is_custom_measurement(internal_name):
```

Comment:
```
    elif (match := MRI_SCHEMA_REGEX.match(internal_name)) is not None:
        return match.group("name")
    ```

Reply:
just did quick test:
```
In [5]: if (match := MRI_SCHEMA_REGEX.match("d:transactions/measurements.speed@millisecond")) is not None:
   ...:     print(match.group("name"))
   ...:
measurements.speed

In [6]:
```
@ahmedetefy wdyt?

Reply:
docs are here https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions

Reply:
Uuu nice, thanks for showing me this feature. Good to know!